codice/ddf
# DDF-4316 Configurable keyword i18n

#### What does this PR do?
- Specific keywords can be configured to be different things. For example, `Sources` can become `Content Stores`. 
- Updates a few marionette components to use JSX instead of handlebars templating. 

#### Who is reviewing it? 
@leo-sakh 
@cantstoptheunk 

#### Select relevant component teams: 
@codice/ui 
@codice/core-apis 

#### Ask 2 committers to review/merge the PR and tag them here.
@andrewkfiedler 
@djblue

#### How should this be tested?
1. Verify that every instance of `Sources` is still `Sources` in Intrigue and the Landing Page
2. `GET https://localhost:8993/search/catalog/internal/config` and verify that this contains an i18n section with a series of keywords like...

```
{
    "source": "source",
    "sources": "sources",
    "Source": "Source",
    "Sources": "Sources"
}
```

#### What are the relevant tickets?
[DDF-4316](https://codice.atlassian.net/browse/DDF-4316)

#### Checklist:

#### Notes on Review Process
Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.
